src/templates/en/2019/chapter.html

```diff
@@ -1,26 +1,36 @@
-{# IMPORTANT!
+{{# IMPORTANT!
```

Comment:
Hi @tyohan, the changes that were made to the `chapter.html` in #222 were so that we can use Node + Showdown + EJS to statically generate the templates - because [mistune wasn't generating tables (amongst other potential problems)](https://github.com/HTTPArchive/almanac.httparchive.org/issues/219#issuecomment-545281951).

If we add the extra curly braces back, it will break the ejs rendering of the markdown. The work done in #232 is supposed to try and help with this, but I'm not sure how best to proceed?

Reply:
@mikegeyser sorry accidentally click re-request review button. 
 
Noted on showdown. Let me try to work around this thing and see if we can combine our works.

Reply:
We should only maintain one markdown parsing script and given that mistune doesn't support markdown tables within HTML, we should deprecate this mistune version in favor of the showdown script @mikegeyser is building.

Sorry that you put so much work into this script and for any confusion as we migrated tools. Having only one script will simplify things a lot.